ui/src/pages/backoffice/Item/ItemDetails/components/ItemCirculation/ItemCirculation.js

```diff
@@ -0,0 +1,206 @@
+import { EmailLink } from '@components';
+import { invenioConfig } from '@config';
+import { MetadataTable } from '@pages/backoffice';
+import { BackOfficeRoutes } from '@routes/urls';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Divider,
+  Grid,
+  Header,
+  Icon,
+  Label,
+  Message,
+  Segment,
+} from 'semantic-ui-react';
+import isEmpty from 'lodash/isEmpty';
+
+class ItemCirculationActiveLoan extends Component {
+  render() {
+    const { data } = this.props;
+
+    const patron = [
+      {
+        name: (
+          <>
+            <Icon name="user" /> Patron{' '}
+          </>
+        ),
+        value: (
+          <>
+            <Link
+              to={BackOfficeRoutes.patronDetailsFor(
+                data.metadata.circulation.patron.id
+              )}
+            >
+              {data.metadata.circulation.patron.name}
+            </Link>
+            <EmailLink
+              email={data.metadata.circulation.email}
+              asButton
+              size="mini"
+              icon
+              labelPosition="left"
+              className="ml-10"
+            >
+              E-mail
+            </EmailLink>
+          </>
+        ),
+      },
+      {
+        name: 'Loan',
+        value: (
+          <Link
+            to={BackOfficeRoutes.loanDetailsFor(
+              data.metadata.circulation.loan_pid
+            )}
+            target={'_blank'}
+          >
+            #{data.metadata.circulation.loan_pid}
+          </Link>
+        ),
+      },
+    ];
+
+    const loan = [
+      {
+        name: 'Loan period',
+        value: `${data.metadata.circulation.start_date} - ${data.metadata.circulation.end_date}`,
+      },
+      { name: 'Extensions', value: data.metadata.circulation.extension_count },
+    ];
+
+    return (
+      <>
+        <Divider horizontal> Loan</Divider>
+        <Grid columns={2}>
+          <Grid.Column width={8}>
+            <MetadataTable labelWidth={5} rows={patron} />
+          </Grid.Column>
+          <Grid.Column width={8}>
+            {' '}
+            <MetadataTable labelWidth={5} rows={loan} />
+          </Grid.Column>
+        </Grid>
+      </>
+    );
+  }
+}
+
+export default class ItemCirculation extends Component {
+  hasActiveLoan = () => {
+    const { data } = this.props;
+
+    return !isEmpty(data.metadata.circulation);
+  };
+
+  isForReference = () => {
+    const { data } = this.props;
+    return invenioConfig.items.referenceStates.includes(data.metadata.status);
+  };
+
+  statusLabel = () => {
+    const { data } = this.props;
```

Comment:
`const { status } = this.props.data.metadata`, since we only use status below

Reply:
fixed